safe_transaction_service/history/services/transaction_service.py

```diff
@@ -276,57 +280,60 @@ def get_all_tx_hashes(
         # the same `execution_date` that the mined tx
         return queryset
 
-    def get_all_txs_from_hashes(
-        self, safe_address: str, hashes_to_search: Sequence[str]
+    def get_all_txs_from_identififers(
+        self, safe_address: str, ids_to_search: Sequence[str]
     ) -> List[Union[EthereumTx, MultisigTransaction, ModuleTransaction]]:
         """
         Now that we know how to paginate, we retrieve the real transactions
 
         :param safe_address:
-        :param hashes_to_search:
+        :param ids_to_search: `SafeTxHash` for MultisigTransactions, `txHash` for other transactions
         :return:
         """
         cached_txs = {
-            hash_to_search: cached_tx
-            for hash_to_search, cached_tx in zip(
-                hashes_to_search,
-                self.get_txs_from_cache(safe_address, hashes_to_search),
+            id_to_search: cached_tx
+            for id_to_search, cached_tx in zip(
+                ids_to_search,
+                self.get_txs_from_cache(safe_address, ids_to_search),
             )
             if cached_tx
         }
-        hashes_not_cached = [
+        id_not_cached = [
             hash_to_search
-            for hash_to_search in hashes_to_search
+            for hash_to_search in ids_to_search
```

Comment:
Another nitpick: do you think `hash_to_search` should be renamed to `id_to_search`?

Reply:
Done! Thank you for all the suggestions @hectorgomezv